manifests/q/QuestSoft/QTranslate/6.8.0.1/QuestSoft.QTranslate.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: QuestSoft.QTranslate
+PackageVersion: 6.8.0.1
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
```

Comment:
Some installers are weeeeird...
![image](https://user-images.githubusercontent.com/15158490/125995241-e32d4fba-5a6a-419e-98f8-271793ed072a.png)
It does require administrator to install tho, so definitively machine scope

Reply:
Saw this too. This is really confusing...